crawl-ref/source/art-data.txt

```diff
@@ -1323,6 +1323,15 @@ TILE:     urand_rift
 TILE_EQ:  rift
 BRAND:    SPWPN_DISTORTION
 
+ENUM:     BATTLE
+NAME:     staff of Battle
+OBJ:      OBJ_WEAPONS/WPN_STAFF
+PLUS:     +0
+COLOUR:   ETC_DISCO
```

Comment:
`ETC_DISCO` is special cased for disco_pan and won't animate in a weapon I don't think (I'd have to code dive).
`ETC_MAGIC` or `ETC_VEHUMET` are probably good choices.